src/Processors/Executors/StreamingFormatExecutor.cpp

```diff
@@ -93,6 +93,12 @@ size_t StreamingFormatExecutor::execute()
         format->resetParser();
         return on_error(result_columns, e);
     }
+    catch (std::exception & e)
```

Comment:
why std exception is enough?

Reply:
It's not, thanks for noticing. I will add processing any exception.

Reply:
Fixed